lib/std/debug.zig

```diff
@@ -507,7 +507,7 @@ pub fn defaultPanic(
                 // ExitData buffer must be allocated using boot_services.allocatePool (spec: page 220)
                 const exit_data: []u16 = uefi.raw_pool_allocator.alloc(u16, exit_msg.len + 1) catch @trap();
                 @memcpy(exit_data, exit_msg[0..exit_data.len]); // Includes null terminator.
-                _ = bs.exit(uefi.handle, .Aborted, exit_msg.len + 1, exit_data);
+                _ = bs.exit(uefi.handle, .Aborted, exit_msg.len + 1, @ptrCast(exit_data));
```

Comment:
Probably doesn't matter much, but I think `exit_data.ptr` would be more readable here.
For `@ptrCast` a reader has to look up the destination type (`?*const anyopaque)` to understand what it does / is used for here.

Reply:
sure, updated